src/main/scala/temple/generate/orchestration/KongConfigGenerator.scala

```diff
@@ -13,7 +13,7 @@ object KongConfigGenerator {
 
   /** Generate a kong service declaration for a Temple service */
   private def generateServiceDef(service: Service): String = {
-    val urls = service.ports map {
+    val urls = service.ports.headOption map {
```

Comment:
If you store a `Ports` type above, you don't have to do this unsafe lookup that depends on ordering